apps/codeforafrica/src/components/NavBarNavList/index.js

```diff
@@ -0,0 +1,72 @@
+import { Link } from "@commons-ui/next";
+import SvgIcon from "@mui/material/SvgIcon";
+import PropTypes from "prop-types";
+import React from "react";
+
+import TwitterIcon from "@/codeforafrica/assets/twitterDesktop.svg";
+import NavList from "@/codeforafrica/components/NavList";
+import NavListItem from "@/codeforafrica/components/NavListItem";
+
+function NavBarNavList({ menu, direction }) {
+  if (!menu?.length) {
+    return null;
+  }
+  return (
+    <NavList direction={direction} sx={{ alignItems: "flex-start" }}>
+      {menu.map((item) => (
+        <NavListItem key={item.label} sx={{ m: "20px" }}>
+          <Link
+            href={item.href}
+            color="inherit"
+            underline="none"
+            sx={{
+              typography: { xs: "h4", md: "subtitle1" },
+              fontWeight: { xs: 700, md: 400 },
```

Comment:
```suggestion
              typography: { xs: "h4", md: "body3" },
```

Should be enough (`body3` = `P3` in Figma).